resdk/resolwe.py

```diff
@@ -94,6 +93,15 @@ def __init__(self, username=None, password=None, url=None):
 
         self.logger = logging.getLogger(__name__)
 
+    def _validate_url(self, url):
+        if not re.match(r'http[s]?://', url):
+            raise ValueError("Please fix url so it starts with https:// or http://.")
```

Comment:
I suggest: "Server url must start with http(s)://"

Reply:
Do we really want to send credentials over http?

Reply:
AFAIK `http` si just for localhost. All real servers have `https`.